examples/dune

```diff
@@ -0,0 +1,35 @@
+; copy test.xpm
+
+(executables
+ (names
+   about accel_tree action assistant assistant_tutorial
+   buttons calc calendar cgets clist combobox combo counter
+   dcalendar dialog-thread drawing ; druid
+   editor2 editor entry2 entrycompletion entry eventbox events2 events expander
+   fifteen filechooser fixed_editor fixpoint
+   gioredirect giotest hello iconview image kaimono
+   label link_button lissajous nihongo notebook
+   pixview pousse progressbar radiobuttons rpn runthread
+   scrolledwin seppala signal_override slide_show socket spin
+   testdnd testgtk testthread timer toolbar tooltip tree tree_model tree_store tron
+ )
+ (modules
+   about accel_tree action assistant assistant_tutorial
```

Comment:
Simpler might be to use `(:standard \ spell)`. If `spell` is the only module to be excluded from this list of course.

Reply:
Thanks @rgrinberg , I guess I'd rather move each package to its own dir, with examples and tests included; but let's see what upstream does say as this would require depending on Dune.

Reply:
The only roadblocker I see to complete a full port is that we cannot enable the build of a library based on the result of the `configurator` program.

I guess we will have to define as many packages as optional dependencies lablgtk can ship.